src/fromager/__main__.py

```diff
@@ -79,6 +79,12 @@
     help="control removal of working files when a build completes successfully",
 )
 @click.option("--variant", default="cpu", help="the build variant name")
+@click.option(
+    "-n",
+    "--number-of-jobs-available",
+    default=1,
```

Comment:
I would prefer to make the default `None`, so it is easier to distinguish "user wants one job" from "user wants default". IIRC `cmake` and `ninja` automatically use all cores unless told otherwise.

```suggestion
    default=1,
```